test/writer/blueprint_test.exs

```diff
@@ -56,6 +82,34 @@ defmodule BlueBird.Test.Writer.BlueprintTest do
     end
   end
 
+  describe "print_attributes/1" do
+    test "prints empty attributes correctly" do
+      assert print_attributes(%{}) == ""
+    end
+
+    test "prints attributes correctly" do
+      assert print_attributes(@complex_body) ==
```

Comment:
Can we put the first `"""` in the assert-line, and then indent the block beginning with `+ Attributes` with two spaces?

Reply:
Of course. But I will change it then on every other place, too. Consistency, you know. :)

Reply:
Thanks 👍